src/main/resources/META-INF/jqassistant-plugin.xml

```diff
@@ -11,6 +11,7 @@
         <resource>spring-mvc.xml</resource>
         <resource>spring-security.xml</resource>
         <resource>spring-transaction.xml</resource>
+        <resource>spring-webservice.xml</resource>
```

Comment:
Please rename to spring-ws.xml (WS is a commonly used abbreviation for SOAP web services)